### ISSUE TITLE
Valid AWS Commands not Working

### ISSUE THREAD

**Issue by kchaudhry-vlocity** [oktadeveloper/okta-aws-cli-assume-role#266]
Hi,

I am running into problems where every command I issue comes back with aws: error: argument operation: Invalid choice, valid choices are:

I just installed the latest version of this tool. Also running latest AWS CLI version.

**Comment by AlainODea** [comment]
1. Did an earlier version work for you?

----
2. Does the AWS CLI work for you independently?

Run this command:

```shell
aws sts get-caller-identity
```

If AWS CLI works it will output something like this:

```json
{
    "UserId": "AROAAAAAAAAAAAAAAAAAA:example@example.com",
    "Account": "123456789012",
    "Arn": "arn:aws:sts::123456789012:assumed-role/EC2Admins/example@example.com"
}
```

If it doesn't, then your AWS CLI install is broken and you'll need to fix that before trying to troubleshoot this tool.

**Comment by AlainODea** [comment]
We have issue templates for a reason. Please use them when you post further issues here. You've provided a very vague description of your issue. I have no idea what specific things you've tried and what specific errors you've encountered, so that makes it quite difficult to help you.

**Comment by kchaudhry-vlocity** [comment]
The aws sts get-caller-identity works fine. Here is more info:

$ aws --version
aws-cli/1.16.89 Python/2.7.15 Darwin/17.7.0 botocore/1.12.79

I just installed the assume role latest version. Push is broken, someone already raised that issue. I have enabled browser=true, and am able to get past that. 

Once the Push is done, I get a list of the accounts. Once I make the selection, the command just throws aws: error: argument command: Invalid choice, valid choices are:

I am running aws s3api list-buckets, which works, okta-aws s3api list-buckets doesn't work.

**Comment by AlainODea** [comment]
Thank you for the added details.

The first argument of okta-aws is a credentials profile name. The tool will create that profile in ~/.aws/credentials. In practice `okta-aws s3api list-buckets` would attempt to run `aws —profile s3api list-buckets` which would result in the error you are seeing.

Does this work?
```shell
okta-aws test s3api list-buckets
```

**Comment by kchaudhry-vlocity** [comment]
Yes that worked. Is there a way to get back to the account listing once I have authenticated? If I want to run the same command against different account for example?
